corehq/apps/domain/deletion.py

```diff
@@ -14,7 +14,7 @@
 from corehq.apps.userreports.dbaccessors import (
     delete_all_ucr_tables_for_domain,
 )
-from corehq.apps.users.audit.change_messages import UserChangeMessage
+from corehq.apps.users.audit.change_messages import UserChangeMessageV1
```

Comment:
So if there is a V2 in future, this class would be `UserChangeMessage` and the new one will be `UserChangeMessageV2`?
Isn't that confusing?

I was thinking it would be `UserChangeMessageV1` and `UserChangeMessageV2` and hence added it proactively so that it would not need to be changed later.
I am fine leaving this just `UserChangeMessage` for now as well. But would like it to have a clear V1 version label through name or file name like APIs in future.

HQ APIs are versioned via files `corehq.apps.api.resources.v0_1.CommCareUserResource` to have clear versions, which is also nice.

Reply:
I don't really feel strongly about it, was just thinking it might be confusing for there to only be a V1 if V2 never ends up existing anyways.  Not a big deal either way

Reply:
I would also expect that on this end you would just change the version of a specific message and you only need to be able to handle multiple versions when interpreting the messages.